Patches/Rimsenal Collection/Enhanced Vanilla/Weapons_Charge_CE.xml

```diff
@@ -287,7 +286,7 @@
 				</AmmoUser>
 				<FireModes>
 				  <aimedBurstShotCount>5</aimedBurstShotCount>
-				  <aiAimMode>AimedShot</aiAimMode>
+				  <aiAimMode>SuppressFire</aiAimMode>
```

Comment:
Not really super concerned, but functionally the assault charger is a tiny heavy charge blaster spec wise, hence why it uses aimed shot for consistency with the mech heavy charge blaster.